libsignal-service/Cargo.toml

```diff
@@ -12,8 +12,8 @@ license = "GPLv3"
 readme = "../README.md"
 
 [dependencies]
-libsignal-protocol = { git = "https://github.com/signalapp/libsignal", tag = "v0.20.0" }
-zkgroup = { git = "https://github.com/signalapp/libsignal", tag = "v0.20.0" }
+libsignal-protocol = { git = "https://github.com/signalapp/libsignal", tag = "v0.21.1" }
+zkgroup = { git = "https://github.com/signalapp/libsignal", tag = "v0.21.1" }
```

Comment:
I'll refrain from merging this until I got the profile code in, because I'll have to update my libsignal fork for Whisperfish too with this bump.